core/templates/dev/head/domain/classifier/PredictionResultObjectFactory.ts

```diff
@@ -31,31 +32,31 @@ oppia.factory('PredictionResultObjectFactory', [function() {
    *   its prediction label. The value is probability (between 0 and 1) that
    *   answer belongs to predicted answer group.
    */
-  var predictionResult = function(label, confidence) {
+  predictionLabel: any;
+  predictionConfidence: any;
+  constructor(label: any, confidence: any) {
     this.predictionLabel = label;
     this.predictionConfidence = confidence;
-  };
-
-  // TODO (ankita240796) Remove the bracket notation once Angular2 gets in.
-  /* eslint-disable dot-notation */
-  predictionResult['createNew'] = function(label, confidence) {
-  /* eslint-enable dot-notation */
-    return new predictionResult(label, confidence);
-  };
-
-  // TODO (ankita240796) Remove the bracket notation once Angular2 gets in.
-  /* eslint-disable dot-notation */
-  predictionResult['getLabel'] = function() {
-  /* eslint-enable dot-notation */
-    return this.predictionLabel;
-  };
+  }
+}
 
-  // TODO (ankita240796) Remove the bracket notation once Angular2 gets in.
-  /* eslint-disable dot-notation */
-  predictionResult['getConfidence'] = function() {
-  /* eslint-enable dot-notation */
+@Injectable({
```

Comment:
Insert new line above.

Reply:
I believe it is already [here](https://github.com/oppia/oppia/pull/7145/files/04d70916960e2a621e192e22f8d3565501fef39e#diff-b334bd1c88a7d3c969c3a9854e7dc8ccR42)